aws/outputs.yaml

```diff
@@ -0,0 +1,60 @@
+Outputs:
```

Comment:
Should not be committed?

Reply:
looks like this is from an earlier commit .. fixed now